tests/js/client/aql/aql-index-cache.js

```diff
@@ -1081,17 +1082,20 @@ function PersistentIndexUniqueReadOnlyStoredValuesSuite() {
   return suite;
 }
 
-jsunity.run(CreateSuite);
-jsunity.run(FiguresSuite);
-jsunity.run(PersistentIndexNonUniqueModifySuite);
-jsunity.run(PersistentIndexUniqueModifySuite);
-jsunity.run(PersistentIndexNonUniqueHugeValueSuite);
-jsunity.run(PersistentIndexNonUniqueReadOnlyCacheDisabledSuite);
-jsunity.run(PersistentIndexNonUniqueReadOnlyCacheEnabledSuite);
-jsunity.run(PersistentIndexNonUniqueReadOnlyStoredValuesSuite);
-jsunity.run(PersistentIndexUniqueReadOnlyCacheDisabledSuite);
-jsunity.run(PersistentIndexUniqueReadOnlyCacheEnabledSuite);
-jsunity.run(PersistentIndexUniqueReadOnlyStoredValuesSuite);
-jsunity.run(OtherIndexesSuite);
+// disable these tests for cluster sanitizer runs since they are very slow
+if (!isCluster || (!versionHas('asan') && !versionHas('tsan'))) {
```

Comment:
No magic execution blocks inside testfiles!
Use filename extensions `-noasan` and `-noncluster`